nix/runtime.nix

```diff
@@ -24,9 +26,9 @@ rec {
       # Postgres will always be accessible via `postgres:5432` from
       # containers.
       port = 5432;
-      user = "ctxlib";
-      password = "ctxlib";
-      db = "ctxlib";
+      user = "ctl";
+      password = "ctl";
+      db = "ctl-${network.name}";
```

Comment:
It does not, because the database is only initialized on the first run. I made this change simply to make it possible to know for which network the DB was initialized.

Reply:
What about

```diff
diff --git a/nix/runtime.nix b/nix/runtime.nix
index 3281f06c6..10d46940c 100644
--- a/nix/runtime.nix
+++ b/nix/runtime.nix
@@ -141,7 +141,7 @@ rec {
             ];
           };
         };
-        postgres = {
+        "postgres-${network.name}" = {
           service = {
             image = "postgres:13";
             ports =
@@ -166,7 +166,7 @@ rec {
               useHostStore = true;
               ports = [ (bindPort datumCache.port) ];
               restart = "on-failure";
-              depends_on = [ "postgres" "ogmios" ];
+              depends_on = [ "postgres-${network.name}" "ogmios" ];
               command = [
                 "${pkgs.bash}/bin/sh"
                 "-c"
@@ -179,7 +179,7 @@ rec {
                     --ogmios-address ogmios \
                     --ogmios-port ${toString ogmios.port} \
                     --db-port 5432 \
-                    --db-host postgres \
+                    --db-host postgres-${network.name} \
                     --db-user "${postgres.user}" \
                     --db-name "${postgres.db}" \
                     --db-password "${postgres.password}" \
```

Reply:
Good idea!